pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>ru.ivi.opensource</groupId>
     <artifactId>flink-clickhouse-sink</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.1.0</version>
```

Comment:
I suggest to set it 1.1.0-SNAPSHOT (if I am not mistaken, deploy-plugin requires it)

Reply:
done